packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyTable.tsx

```diff
@@ -116,37 +108,25 @@ export const CalculatedPropertyTable = ({
     [onClickModify]
   );
 
+  const deleteProperty = useCallback(async (iModelId: string, accessToken: string, propertyId: string) => {
```

Comment:
The signature of this function would look nicer if it only accepted `propertyId`. I see no point requiring `iModelId` and `accessToken` if `groupId` and `mappingId` are captured.

Reply:
I'm not against it. I will just have to bring `const { getAccessToken, iModelId } = useGroupingMappingApiConfig();` to every table to capture it. Is that ok?

Reply:
Yes. The signature would make more sense.

Reply:
Updated signature.